qutip/hsolve.py

```diff
@@ -31,8 +31,10 @@
 #    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 """
-This module provides exact solvers for a system-bath setup using the
-hierachical method.
+2016-03-9: This module has been deprecated.
```

Comment:
Good point. No, hsolve is new since 3.1. It's unlikely that many users
would be affected anyway
I will delete the old module and name the new one in nonmarkov 'hsolve'
On 11 Mar 2016 05:15, "Robert Johansson" <notifications@github.com> wrote:

> In qutip/hsolve.py
> <https://github.com/qutip/qutip/pull/455#discussion_r55787789>:
>
> > @@ -31,8 +31,10 @@
> >  #    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
> >  ###############################################################################
> >  """
> > -This module provides exact solvers for a system-bath setup using the
> > -hierachical method.
> > +2016-03-9: This module has been deprecated.
>
> The qutip.hsolver has not yet been part of a release, right? Or was it
> already here in 3.1? If not, then you should not have deprecation message.
> Just deleting this module is fine. We dont guarantee api stability until
> after release.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qutip/qutip/pull/455/files#r55787789>.
>